content/pt/docs/languages/java/instrumentation.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Ecossistema de Instrumentação
+aliases:
+  - /docs/java/getting_started
+  - /docs/java/manual_instrumentation
+  - manual
+  - manual_instrumentation
+  - libraries
+weight: 10
+description: Ecossistema de Instrumentação no OpenTelemetry Java
+default_lang_commit: d65798034935fcfdbdc6599b6e6d3dd942fbe62c
+cSpell:ignore: Logback logback
+---
+
+<!-- markdownlint-disable no-duplicate-heading -->
+
+A instrumentação registra a telemetria usando a [API](../api/). O [SDK](../sdk/)
+é a implementação de referência embutida na API, e é
+[configurada](../configuration/) para processar e exportar a telemetria
+produzida pelas chamadas de instrumentação da API. Esta página discute o
+ecossistema de instrumentação no OpenTelemetry Java, incluindo recursos para
+usuários finais e tópicos relacionados à instrumentação:
+
+- [Categorias de instrumentação](#instrumentation-categories): Existem diversas
+  categorias de instrumentação para diferentes casos de uso e padrões de instalação.
+- [Propagação de Contexto](#context-propagation): Propagação de Contexto provê
+  uma correlação entre rastros, métricas, e logs, permitindo que os sinais
+  se complementem.
+- [Convenções semânticas](#semantic-conventions): As convenções semânticas definem
+  como produzir telemetria para operações padrão.
+- [Log instrumentation](#log-instrumentation): A Convenção semântica define como
+  produzir telemetria para operações padronizadas.
```

Comment:
```suggestion
- [Log instrumentation](#log-instrumentation)
```

Pra poder dar merge nesse PR, vamos tirar a descrição, melhor do que ficar a errada, assim quando a versão em inglês for corrigida vc pode criar outro PR